lib/position.dart

```diff
@@ -87,6 +87,49 @@ class Position {
     return this;
   }
 
+  /// Rotate around origin; [angle] in degrees.
+  Position rotateDeg(double angle) {
+    return rotate(angle * math.pi / 180);
+  }
+
+  /// Change current rotation by delta angle; [angle] in radians.
+  Position rotateDelta(double radians) {
+    return rotate(angle() + radians);
+  }
+  
+  /// Change current rotation by delta angle; [angle] in degrees.
+  Position rotateDeltaDeg(double degrees) {
+    return rotateDeg(angleDeg() + degrees);
+  }
+
+  /// Returns the angle of this vector from origin in radians.
+  double angle() {
+    return math.atan2(y, x);
+  }
+
+  /// Returns the angle of this vector from origin in degrees.
+  double angleDeg() {
+    return angle() * 180 / math.pi;
+  }
+
+  /// Returns the angle of this vector from another [Position] in radians.
+  double angleFrom(Position other) {
+    //Technically the origin is not a vector so you cannot find the angle between
+    //itself and another vector so just default to calculating the angle between
+    //the non-origin point and the origin to avoid division by zero
+    if(x == 0 && y == 0)
+      return other.angle();
+    if(other.x == 0 && other.y == 0)
+      return angle();
```

Comment:
just for linter compliance:
1. comments `// xxx` (space)
2. `if (xxx) {` (space & add braces)

Reply:
Done!